myia/operations/prim_handle.py

```diff
@@ -1,14 +1,9 @@
 """Definitions for the primitive `handle`."""
 
-from ..lib import AbstractHandle, HandleInstance, standard_prim
+from ..lib import AbstractHandle, standard_prim
 from . import primitives as P
 
 
-def pyimpl_handle(init):
-    """Implement `handle`."""
-    return HandleInstance(init)
```

Comment:
Why are you removing this?

Reply:
Because it's not called anymore.  Unless you want me to add a DebugVM test to keep it.

Reply:
The Universe feature won't work with the DebugVM anymore, so leave it removed, I suppose.